examples/practice-typescript/src/components/TableHeader /index.tsx

```diff
@@ -0,0 +1,40 @@
+import User from 'scripts/types/user';
+import './index.scss';
+import Product from 'scripts/types/product';
+
+interface HeaderItemProps {
+  label: string;
+}
+
+const HeaderItem = ({ label }: HeaderItemProps) => {
+  const formatLabel = (str: string) => {
+    return str
+      .replace(/\b\w/g, (char) => char.toUpperCase())
+      .replace(/([A-Z])/g, ' $1')
+      .trim();
+  };
+
+  return <th className="table-head">{formatLabel(label)}</th>;
+};
+
+interface TableHeaderProps {
+  headers: User[] | Product[];
```

Comment:
Should be `headerItems`

Reply:
Can we provide generic type here instead `User[] | Product[]`

Reply:
Done